.travis.yml

```diff
@@ -84,6 +124,12 @@ jobs:
           env: OPENSSL_VER=1.1.0l OPENSSL_PATCH_VER=1.1.0d
         - <<: *linux-s390x
           env: OPENSSL_VER=1.1.1k OPENSSL_PATCH_VER=1.1.1f
+        - <<: *linux-ppc64le
```

Comment:
I don‘t think there is a need to test openssl 1.0.2 and 1.1.0 anymore.

Reply:
Ok, so the below SSL version jobs are no longer needed?

```
    - OPENSSL_VER=1.0.2u OPENSSL_PATCH_VER=1.0.2h
    - OPENSSL_VER=1.1.0l OPENSSL_PATCH_VER=1.1.0d
```
I can remove the corresponding jobs if not needed.